src/legacy/core_plugins/kibana/server/routes/api/management/saved_objects/find.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * This file wraps the saved object `_find` API and is designed specifically for the saved object
+ * management UI. The main difference is this will inject a root `meta` attribute on each saved object
+ * that the UI depends on. The meta fields come from functions within uiExports which can't be
+ * injected into the front end when defined within uiExports. There are alternatives to this but have
+ * decided to go with this approach at the time of development.
+ */
+
+import Joi from 'joi';
+import { injectMetaAttributes } from '../../../../lib/management/saved_objects/inject_meta_attributes';
+
+export function registerFind(server) {
```

Comment:
Can you add a comment explaining what this route does differently than the regular saved objects find route?

Reply:
Easy to miss with the license header, there is one from line 20 to 26, maybe it should be moved down?

Reply:
Ahh no it's fine, just missed it 👍